assets/css/style.css

```diff
@@ -184,7 +184,7 @@ body > header nav > .menu, body > footer nav > .menu {
 
 body > header nav > .menu > ul, body > footer nav > .menu > ul {
 				position: absolute;
-				z-index: 2;
+				z-index: 1;
```

Comment:
This seems unrelated?

Reply:
Yes, sorry. This (and the change in `package-lock.json`) were updated when I ran `npm run build` locally. I can revert them if you prefer?